pkg/debugutil/pprof.go

```diff
@@ -17,10 +17,16 @@ package debugutil
 import (
 	"net/http"
 	"net/http/pprof"
+	"runtime"
 )
 
 const HTTPPrefixPProf = "/debug/pprof"
 
+func init() {
+	// 1 out of 5 mutex events are reported, on average
+	runtime.SetMutexProfileFraction(5)
```

Comment:
This will enable mutex profiling even if it's not exposed via http.

Calling it in `PProfHandlers` would avoid the unconditional enable. Also, maybe also check if it's already been set so it won't clobber any existing settings:
```go
if runtime.SetMutexProfileFraction(-1) != 0 {
    // 1 out of 5 mutex events are reported, on average
    runtime.SetMutexProfileFraction(5)
}
```